server/src/main/java/org/candlepin/resteasy/filter/VerifyAuthorizationFilter.java

```diff
@@ -169,41 +183,74 @@ protected Owner findOwner(Map<Verify, Object> argMap, Principal principal, Acces
             }
 
             if (Owner.class.isAssignableFrom(verifyType)) {
-                Access requiredAccess = defaultAccess;
```

Comment:
Once findOwnerFromParams returns, the next thing we do after putting the owner in the context is call hasAccess, which does those checks as well. If the access check context differs (I don't believe it does), then that may have been removed in error.